frontend-react/cypress.config.ts

```diff
@@ -12,9 +13,36 @@ const cypressConfig = require(path.resolve(
     `./cypress/cypress.${cypressEnv}.json`
 ));
 
+function getEnvOrDefault(name: string, dfault?: string) {
```

Comment:
Ah, minor typo: `default`

Reply:
default is a reserved word 😛

Reply:
Hahaha oh duh.  Can we make it something like `defaultValue` instead then?